cmd/controller-manager/app/controllermanager.go

```diff
@@ -12,7 +12,7 @@ import (
 	"k8s.io/client-go/dynamic"
 	"k8s.io/component-base/logs"
 	"k8s.io/klog/v2"
-	"sigs.k8s.io/controller-runtime"
+	controllerruntime "sigs.k8s.io/controller-runtime"
```

Comment:
This might have been fixed by #322 this morning. Please rebase your code.

Reply:
gotcha